worker/src/tests_polkadex_DB.rs

```diff
@@ -0,0 +1,115 @@
+use std::{thread, time};
+
+use polkadex_primitives::types::{Order, OrderSide, OrderType, SignedOrder};
+
+use crate::polkadex_db::{KVStore, RocksDB};
+
+#[test]
+fn test_db_initialization() {
+    assert!(RocksDB::initialize_db(true).is_ok());
+    assert!(RocksDB::load_orderbook_mirror().is_ok());
+}
+
+#[test]
+fn test_write_and_delete() {
+
```

Comment:
Ideally, each unit test can be run independent of the other tests. Can the DB not be initialized twice?

Reply:
The current implementation takes the DB initialized to a fixed filename ( defined in constants). Also, initializing multiple DB instances will require multiple pointers. Due to this reason, the unit tests are dependent. What do you think?

Reply:
I am moving this Conversation as a separate issue. Since I don't want to wait for this till Monday, I am merging this PR to work on Gateway.